src/bin/cql-stress-cassandra-stress/java_generate/distribution/normal.rs

```diff
@@ -56,3 +57,99 @@ impl Distribution for NormalDistribution {
         self.rng.get().set_seed(seed as u64)
     }
 }
+
+pub struct NormalDistributionFactory {
+    min: i64,
+    max: i64,
+    mean: f64,
+    standard_deviation: f64,
+}
+
+impl NormalDistributionFactory {
+    fn new(min: i64, max: i64, mean: f64, standard_deviation: f64) -> Result<Self> {
+        NormalDistribution::verify_args(min, max, standard_deviation)?;
+        Ok(Self {
+            min,
+            max,
+            mean,
+            standard_deviation,
+        })
+    }
+}
+
+impl DistributionFactory for NormalDistributionFactory {
+    fn create(&self) -> Box<dyn Distribution> {
+        Box::new(
+            NormalDistribution::new(self.min, self.max, self.mean, self.standard_deviation)
+                .unwrap(),
+        )
+    }
+}
+
+impl NormalDistributionFactory {
+    fn do_parse_from_description(desc: &Description<'_>) -> Result<Box<dyn DistributionFactory>> {
+        // See https://github.com/scylladb/scylla-tools-java/blob/master/tools/stress/src/org/apache/cassandra/stress/settings/OptionDistribution.java#L202.
+        let mut iter = desc.args.iter();
+
+        let (min, max) = match (iter.next(), iter.next()) {
+            (Some(min), Some(max)) => (min.parse::<i64>()?, max.parse::<i64>()?),
+            _ => anyhow::bail!("Expected at least 2 arguments"),
+        };
```

Comment:
Nit: we already have `Description::check_argument_count`, so maybe we could have a method which checks that we have at least N arguments as well.